features/other_b/subscriptions.feature

```diff
@@ -389,3 +389,23 @@ Scenario: subscriptions are not deleted without confirmation
   When I go to the subscriptions page for "second_user"
   Then I should see "My Subscriptions"
     And I should see "Awesome Story (Work)"
+
+  # NOTE: currently, posting a second chapter to a draft work will post both chapters
+  Scenario: Posting a second chapter to a draft work should notify subscribers about the work being posted and chapter 2
+    Given I am logged in as "first_user"
+      And "second_user" subscribes to author "first_user"
+      And all emails have been delivered
+    When I am logged in as "first_user"
+      And I set up the draft "Half and Half"
+      And I press "Save Draft"
+      And I follow "My Dashboard"
+      And I follow "Drafts ("
```

Comment:
I think it's clearer to use `I view the work "Half and Half"` instead of going from the drafts page.